packages/webpack/src/migrations/update-15-6-3/__snapshots__/webpack-config-setup.spec.ts.snap

```diff
@@ -0,0 +1,63 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`15.6.3 migration (setup webpack.config file) should create webpack.config.js file if it does not exist 1`] = `
+"
+        const { composePlugins, withNx } = require('@nrwl/webpack');
+        const { withReact } = require('@nrwl/react');
+      
+        // Nx plugins for webpack.
+        module.exports = composePlugins(withNx(), withReact(), (config) => {
+          // Update the webpack config as needed here.
+          // e.g. config.plugins.push(new MyPlugin())
+          return config;
+        });
+        "
+`;
+
+exports[`15.6.3 migration (setup webpack.config file) should create webpack.config.js file if it does not exist 2`] = `
+"
+        const { composePlugins, withNx } = require('@nrwl/webpack');
+        const { withReact } = require('@nrwl/react');
+      
+        // Nx plugins for webpack.
+        module.exports = composePlugins(withNx(), withReact(), (config) => {
+          // Update the webpack config as needed here.
+          // e.g. config.plugins.push(new MyPlugin())
+          return config;
+        });
+        "
+`;
+
+exports[`15.6.3 migration (setup webpack.config file) should rename existing webpack.config file and create new one that requires it 1`] = `
+"
+        const { composePlugins, withNx } = require('@nrwl/webpack');
+        const { withReact } = require('@nrwl/react');
+
+        // Nx plugins for webpack.
+        module.exports = composePlugins(withNx(), withReact(), (config) => {
+          // Note: This was added by an Nx migration.
+          // You should consider inlining the logic into this file.
+          return require('./webpack.config.js')(config, context);
```

Comment:
Where does `context` come from? It seems like a missing identifier or arg.

Reply:
Ha, good catch, thanks! :D I'll discuss with @jaysoo ! I should mark this as draft in any case, it still needs some tweaks!